tools/js.py

```diff
@@ -0,0 +1,48 @@
+#    Copyright 2022 Cryspen Sarl
+#
+#    Licensed under the Apache License, Version 2.0 or MIT.
+#    * http://www.apache.org/licenses/LICENSE-2.0
+#    * http://opensource.org/licenses/MIT
+
+import os
+import subprocess
+import sys
+import shutil
+from os.path import join as path_join
+from os.path import exists as path_exists
+
+
+def build_js():
+    """Build the JS bindings."""
+
+    cwd = path_join(os.path.dirname(os.path.realpath(__file__)), "..")
+    src_path = path_join(cwd, "src/wasm")
```

Comment:
This isn't portable. But you can do
```suggestion
    src_path = path_join(cwd, "src", "wasm")
```